{{ cookiecutter.project_name }}/src/flows.py

```diff
@@ -8,11 +8,11 @@ def uppercase_the_text(text: str) -> str:
 
 
 @flow(log_prints=True)
-def example_flow(text: str = "Hi"):
+def main(text: str = "Hi"):
```

Comment:
Die naam `main` komt als naam van de flow in de prefect interface terecht volgens mij.
Ik ben nu met een PR bezig waarin ik de flows prefix met de projectnaam om ze in prefect uit elkaar te kunnen houden.

main: er kunnen meerdere flows zijn. De `if name == main` is er nu alleen voor wat lokaal testen, dus het hoeft niet officieel main te heten.